### DISCUSSION TITLE
Add low-spec mode for low-spec device

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#22358] [Ideas] [answered]
I have a low-spec Android tablet and I want to play osu!lazer on it, but it is very laggy, fps is only about 20fps,even with the lowest setting. I also have a high-spec phone but it's too small to play, so i want to add low-spec mode to osu!laser

**Comment by Theighlin** [comment, marked as answer]
Duplicate of #19091. Short answer: wait for optimisations.
